citi-onboarding/Delta-odonto
# feature/banner

## What I did
Created the model of the banner
Added 'Banner' to the index file

## How to test
Please, follow the next steps

1- On the terminal, access the folder "server" and then run the "index.js" file with node by typing the following commands:
`cd server`
`node index.js`

2- Open another terminal,  access the folder "client" and then run the "index.js" file with node by typing the following commands:
`cd client `
`yarn server`

3- Type the adress "localhost:3001/admin/" on your browser then the Keystone interface will appear.

Log in with this:
email: admin@gmail.com
password: admin

(This piece of information can also be found on server>updates>0.0.1-admin.js , you can click on the folders and file or type on the terminal:
`cd server`
`cd updates`
and you will find the "0.0.1-admin.js" file.)

4- Now you will be able to see and edit the Banner section.
You will be able to create a banner and change its value proposition and value proposition description and image.
